doc/changelog.rst

```diff
@@ -1,6 +1,7 @@
 Changelog
 =========
 
+* :feature:`467` Automatically detect yielded fixtures
```

Comment:
"Slash now automatically detects yielded fixtures, and using `@yield_fixture` explicitly is no longer required"